locales/en/yourloops.json

```
@@ -391,7 +392,8 @@
   "today": "Today",
   "tooltip-release-notes": "Release notes",
   "training": "Training",
-  "training-body": "New <trainingLink>{{training}}</trainingLink> available, please read what's new before continuing on yourloops. You can also read it later, by clicking in the {{training}} footer link.",
+  "training-body": "New {{training}} available, please read what's new before continuing on yourloops.",
```

`{{training}}` variable could be removed too (also in the component)
```suggestion
  "training-body": "New training available, please read what's new before continuing on yourloops.",
```